### DISCUSSION TITLE
[灌水] 又和前辈去健身房了

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#2457] [其他]
![mmexport1723818686556](https://github.com/user-attachments/assets/8064595d-1f31-4b93-b415-8977c98bac8e)
山狮案结束以后终于有时间和前辈一起来健身房了。但没想到居然在更衣室的时候遇到了铃店长，手里还端着随身携带的相机。前辈看到以后居然还请店长帮我们拍健完身的合照，说什么有纪念价值（现在谈纪念还太早了吧）……真是的，前辈是机器人不会流汗，但我全身都被汗水浸湿了，这样拍出来的照片也太…啊啊啊好羞耻！！

> 作者：pixiv SH339 （都是像素画风的作品哦）

**Comment by dolokong** [comment]
机器人也会通过人类的常规锻炼增强机能吗

**Reply by ZhuYuan-148** [reply]
应该是为了防止战斗部件生锈…吧

**Comment by mingyu818** [comment]
NB了，哥

**Comment by GanlinSama** [comment]
好伟大的朱鸢警光<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by NekoWise** [comment]
哪家健身房，有时间还不如去哩查茶奶来呗茶奶呢

**Reply by ZhuYuan-148** [reply]
谢谢这位小妹妹的推荐，但我喝前辈泡的茶就行了，茶奶有点甜。

**Comment by duskbirds** [comment]
难怪朱鸢身材这么好。我是不是也应该去呢？想成为安东那样的强壮猛男，但朋友说我太瘦了，先把体重提上去。<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Reply by ZhuYuan-148** [reply]
首先要注意营养均衡，当然健身也是很好的习惯。只要肯坚持，一定能做到的。

**Comment by yutong903** [comment]
朱鸢警官你屁股上的把手呢？<img src="https://img-static.mihoyo.com/communityweb/upload/dce2b6d619041e9f7ad133c804fb1a0d.png" alt="妮可_震惊" width="75" >

**Reply by ZhuYuan-148** [reply]
嗯？不出勤的时候当然要卸下来……不对…这是绳降工具啊喂！

**Comment by aqswzxgao** [comment]
警官也能进绳网？<img src="https://img-static.mihoyo.com/communityweb/upload/dce2b6d619041e9f7ad133c804fb1a0d.png" alt="妮可_震惊" width="75" >

**Comment by HoshimiMiyabi120** [comment]
老同学，有机会的话，我还想和你同行一天，毕竟「凭借同学录回忆起所有同学的名字」也是修行中的一环
